### ISSUE TITLE
Bernoulli Logo?

### ISSUE THREAD

**Issue by noahblon** [nerdery-bernoulli/wiki#2]
I've been thinking it'd be cool to have a logo for our group - something that would look sweet on a t-shirt.  Anyway, I did some research on Bernoulli and found a couple of interesting things:
- Bernoulli was obsessed with what he called the "spira mirabilis" - the magnificent spiral.  He was particularly taken with the [logarithmic spiral which is expressed in nature as fractals](https://en.wikipedia.org/wiki/Logarithmic_spiral), and also in the [golden ratio](https://en.wikipedia.org/wiki/Golden_spiral), which is something commonly referred to in web design.
- Bernoulli is famous for his work with epicycloids, [which is something like what a spirograph draws](https://en.wikipedia.org/wiki/Epicycloid). 

I think both of these ideas could be cool to base a logo on.  Any other ideas out there?  If you want to help out design a logo, lemme know.

**Comment by kruegkev** [comment]
I've always wanted a logo and had meant to design one a long time ago. I'd
have fun running with some of the ideas you just suggested. Or we could
make it more of a collaborative thing.

On Wed, Sep 2, 2015 at 1:17 PM, Noah Blon notifications@github.com wrote:

> I've been thinking it'd be cool to have a logo for our group - something
> that would look sweet on a t-shirt. Anyway, I did some research on
> Bernoulli and found a couple of interesting things:
> - Bernoulli was obsessed with what he called the "spira mirabilis" -
>   the magnificent spiral. He was particularly taken with the logarithmic
>   spiral which is expressed all across nature in fractals
>   https://en.wikipedia.org/wiki/Logarithmic_spiral, and also in the golden
>   ratio https://en.wikipedia.org/wiki/Golden_spiral, which is
>   something commonly referred to in web design.
> - Bernoulli is famous for his work with epicycloids, which is
>   something like what a spirograph draws
>   https://en.wikipedia.org/wiki/Epicycloid.
> 
> I think both of these ideas could be cool to base a logo on. Any other
> ideas out there? If you want to help out design a logo, lemme know.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nerdery-bernoulli/wiki/issues/2.

## 

Kevin Krueger
Client-Side Developer
The Nerdery
(651) 341-9429 // mobile

**Comment by noahblon** [comment]
Yeah, everyone is welcome to come up with a design.  It would be cool if it the logo was something that could be animated with code in a cool way.

September 23rd will be our cutoff for submissions after which we can vote.  Post em up here, in progress if you want feedback, or when your finished.

**Comment by shannonmoeller** [comment]
Side-by-side red bull cans (with a small gap between)?

https://www.youtube.com/watch?v=P-xNXrELCmU

**Comment by kruegkev** [comment]
Unfortunately, that's the wrong Bernoulli. There's been a lot of confusion
since the origin of our team.

Here's our man: https://en.wikipedia.org/wiki/Jacob_Bernoulli

On Thu, Sep 3, 2015 at 11:48 AM, Shannon Moeller notifications@github.com
wrote:

> Side-by-side red bull cans?
> 
> https://www.youtube.com/watch?v=P-xNXrELCmU
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nerdery-bernoulli/wiki/issues/2#issuecomment-137508169
> .

## 

Kevin Krueger
Client-Side Developer
The Nerdery
(651) 341-9429 // mobile

**Comment by shannonmoeller** [comment]
whoops

**Comment by shannonmoeller** [comment]
> Bernoulli chose a figure of a logarithmic spiral and the motto Eadem mutata resurgo ("Changed and yet the same, I rise again") for his gravestone; the spiral executed by the stonemasons was, however, an Archimedean spiral, “[Jacques Bernoulli] wrote that the logarithmic spiral ‘may be used as a symbol, either of fortitude and constancy in adversity, or of the human body, which after all its changes, even after death, will be restored to its exact and perfect self’.” (Livio 2002: 116).
> 
> https://en.wikipedia.org/wiki/Jacob_Bernoulli

![image](https://cloud.githubusercontent.com/assets/155164/9683893/0d3bc0a8-52e2-11e5-89cc-b235997374c5.png)

**Comment by shannonmoeller** [comment]
Tweaked to be a little more consistent with the Lovelace logo.

![image](https://cloud.githubusercontent.com/assets/155164/9684243/3cd959ea-52e4-11e5-81aa-65ef627f7fa9.png)

**Comment by lwloszek** [comment]
I like it!  I have a few little pieces of feedback: 
- Match font
- Square off the teeth on the gear
- Maybe use a spiral that is closer to the golden ratio or try it with the epicycloid?

**Comment by shannonmoeller** [comment]
1. I used the font from the Boilerplate GUI.
2. Yeah, not happy with the teeth (and really not happy with the teeth in the lovelace logo). I want them to be closer to real gear teeth. Just didn't take the time on the prototype.
3. The spiral was made with the Illustrator spiral tool which creates logarithmic spirals, as Bernoulli requested.

**Comment by noahblon** [comment]
Alright, new deadline for the logo submissions is Wednesday 9/30.  That is a hard cutoff (mostly to force me to do one!)

**Comment by noahblon** [comment]
OK, I took a shot!
![screenshot 2015-09-30 at 11 18 09 pm](https://cloud.githubusercontent.com/assets/561590/10212775/a13ddb0a-67c9-11e5-8aa2-56ecab9b6b5f.png)

**Comment by lwloszek** [comment]
I kinda love the little elephant.

I wonder if there is a way we can combo the Latin into this one.

**Comment by noahblon** [comment]
Ha! I actually thought about adding a little eye and some tusks.  As far as working more on it, lets have a vote first!  The winning entry can be tweaked further if desired.  Add +1 for the logo you like the most.  Unless everyone votes, voting ends next Wednesday.

**Comment by dpren** [comment]
+1 for Noah's monochrome spiral :+1:

What is this about an elephant?

**Comment by lwloszek** [comment]
I would also like the elephant.  :elephant: 

@dpren https://s-media-cache-ak0.pinimg.com/736x/d2/dc/41/d2dc4105e03a5305d6822599eac7d4de.jpg

**Comment by lwloszek** [comment]
I also kinda like the idea of having another 1/2 of the circle be the integration of the cog in Shannon's design.

**Comment by shannonmoeller** [comment]
See. We needed more options. +1 :elephant: 

I would like to see us use the official Nerdery font, or the Boilerplate webfont-able variant.

**Comment by shannonmoeller** [comment]
Gear teeth and font change? (needs cleanup to get rid of the artifacts)

![image](https://cloud.githubusercontent.com/assets/155164/10286154/61668ba0-6b5a-11e5-99cd-71d7aa41e686.png)

**Comment by shannonmoeller** [comment]
Spiral gears are a thing: https://youtu.be/AlQBh22Qzd0?t=12

**Comment by noahblon** [comment]
OK! Voting is closed.  So it seems the consensus is to do some more work on this.  SO!  Another week for suggestions/tweaks and we'll vote again?  Anyone can jump in now if they are interested.
